kubernetes/README.md

```diff
@@ -1,41 +1,63 @@
-# Invidious Helm chart
+# Invidious Kubernetes Deployment
```

Comment:
This whole file should go in the documentation in my opinion.

I'll let @unixfox decide if that's the case (and review this PR).